net-misc/gsasl/gsasl-1.8.0.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="The GNU SASL tool"
+DESCRIPTION="GNU SASL is an implementation of the Simple Authentication and \
+Security Layer framework and a few common SASL mechanisms. SASL is used by \
+network servers (e.g., IMAP, SMTP) to request authentication from clients, and \
+in clients to authenticate against servers.
+
+Supported mechanisms are ANONYMOUS, EXTERNAL, LOGIN, PLAIN, SECURID, NTLM, \
+DIGEST-MD5, CRAM-MD5, SCRAM-SHA-1, SCRAM-SHA-1-PLUS, GS2-KRB5, GSSAPI."
+HOMEPAGE="https://www.gnu.org/software/gsasl/"
+COPYRIGHT="2004-2012 Simon Josefsson"
+LICENSE="GNU GPL v3
+	GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://ftp.gnu.org/gnu/gsasl/gsasl-$portVersion.tar.gz"
+CHECKSUM_SHA256="310262d1ded082d1ceefc52d6dad265c1decae8d84e12b5947d9b1dd193191e5"
+PATCHES="gsasl-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	gsasl = $portVersion
+	cmd:gsasl = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libgsasl
+	"
```

Comment:
I saw this in `config.log`:
```
configure:17001: WARNING: GnuTLS not found.  STARTTLS support disabled.
configure:17004: checking if STARTTLS support via GnuTLS should be built
configure:17006: result: no
```
I noticed that the it also checks the availability of libiconv and libintl, so maybe we could also add all this stuff:
```diff
 REQUIRES="
 	haiku
 	lib:libgsasl
+	lib:libgnutls
+	lib:libiconv
+	lib:libintl
 	"
```
(I don't know if this is a good idea, I'm trying it right now.)

Reply:
Oops, sorry, I should have placed libgnutls above libgsasl, not bellow.
Could you please swap these 2 lines, so that all `lib:lib*` lines be sorted alphabetically? Thanks!